src/components/views/right_panel/UserInfo.tsx

```diff
@@ -1038,7 +1038,7 @@ const PowerLevelEditor: React.FC<{
 
             const [confirmed] = await finished;
             if (!confirmed) return;
-        } else if (myUserId === target) {
+        } else if (myUserId === target && powerLevel !== 100) {
```

Comment:
It's possible to start with a power level greater than 100, so I think rather than just comparing it to 100, we should check whether the power level went down by comparing it to the original level.